util/domainutil/domain_vars.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2019 PingCAP, Inc.
```

Comment:
The file is all about repairing table, move it to another dictionary and rename it.

Reply:
Addressed.